mw-server/db/query/ways.query.sql

```diff
@@ -240,3 +240,47 @@ RETURNING *,
 -- name: DeleteWay :exec
 DELETE FROM ways
 WHERE uuid = $1;
+
+-- name: GetWayDetailsById :many
```

Comment:
why do we need another one method? Why can't we just improve old one?

Reply:
done